### PR TITLE
Document the enabling of a higher thread priority in Servo

### PR DESCRIPTION
**Pull request by AndyZe**
### Description

This describes how to set permissions for Servo to use a higher thread priority. Basically this is a port of this `ros2_control` PR:

https://github.com/ros-controls/ros2_control/pull/781

Merge with this MoveIt2 PR:

https://github.com/ros-planning/moveit2/pull/1464

### CODE REVIEW

File: doc/examples/realtime_servo/realtime_servo_tutorial.rst [hunk 1]
```diff
@@ -51,6 +53,35 @@ Expected Output
...
+Increase Thread Priority if a Realtime Kernel is Installed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For best performance when controlling hardware you want the main servo loop to have as little jitter as possible.
+The normal linux kernel is optimized for computational throughput and therefore is not well suited for hardware control.
+The two easiest kernel options are the `Real-time Ubuntu 22.04 LTS Beta <https://ubuntu.com/blog/real-time-ubuntu-released>`_ or `linux-image-rt-amd64 <https://packages.debian.org/bullseye/linux-image-rt-amd64>`_ on Debian Bullseye.
+
+If you have a realtime kernel installed, the main thread of ``servo_calcs`` automatically attempts to configure ``SCHED_FIFO`` with a priority of ``50``.
+By default, the user does not have permission to set such a high priority.
+To give the user such permissions, add a group named realtime and add the current user to this group:
```
[the remark this answers is not in the document]
Comment (AndyZe) [reply doc/examples/realtime_servo/realtime_servo_tutorial.rst:65]: Disagree -- the ros2_control version of this was mostly copied from a Panda repo. So... should I just link to the Panda instructions? That doesn't make sense.

Also I need to update the priority to 40 here.

[the remark this answers is not in the document]
Comment (AndyZe) [reply doc/examples/realtime_servo/realtime_servo_tutorial.rst:65]: It's not much text, anyway.

[the remark this answers is not in the document]
Comment (tylerjw) [reply doc/examples/realtime_servo/realtime_servo_tutorial.rst:65]: The original part I wrote was not copy-pasted. I did not realize you had copy-pasted that other text. I think we should update it to just refer to the panda repo then.

[the remark this answers is not in the document]
Comment (AndyZe) [reply doc/examples/realtime_servo/realtime_servo_tutorial.rst:65]: I appreciate the suggestion but I'm not going to make the change. For one reason, the text here is tuned to apply to Servo in a few ways. Also, I think people are less likely to read a series of links, and the text we link to may change.

File: doc/examples/realtime_servo/realtime_servo_tutorial.rst [hunk 2]
```diff
@@ -13,10 +13,12 @@ This tutorial shows how to use MoveIt Servo to send real-time servo commands to
 
 Getting Started
 ---------------
+
 If you haven't already done so, make sure you've completed the steps in :doc:`Getting Started </doc/tutorials/getting_started/getting_started>`.
 
 Launching a Servo Node
 ----------------------
+
```
Comment (tylerjw) [inline doc/examples/realtime_servo/realtime_servo_tutorial.rst:21]: Also, what is with these new-lines? Do you have some feature of your editor auto-formatting this?

If this feature is generally useful you should add it to pre-commit with a separate change.

**Comment by sjahr** [comment]
@ibrahiminfinite @AndyZe Is this PR outdated by the servo refactoring?

**Comment by ibrahiminfinite** [comment]
This PR has useful info.
And the thread priority is now a proper parameter which can be set.
So it would be great to have some documentation about it.

@AndyZe do you want to update this PR or should  I add it in a PR along with other upcoming changes to the docs?

**Comment by AndyZe** [comment]
I'll update. Thanks guys.

File: doc/examples/realtime_servo/realtime_servo_tutorial.rst (hunk 2 above)
Comment (AndyZe) [reply doc/examples/realtime_servo/realtime_servo_tutorial.rst:21]: Those were manual changes.

**Comment by AndyZe** [comment]
Easiest just to close this PR and start a new one

**Pull request closed without merging**